README.md

```diff
@@ -93,67 +108,6 @@ import { PuikButton } from '@prestashopcorp/puik'
 
 <p align="right">(<a href="#top">back to top</a>)</p>
 
-#### Full import
-
-If you don't care about bundle size you can full import the library by following these instructions
-
-Import the vue library and the css directly into your main.js / main.ts
-
-```typescript
-import { createApp } from 'vue'
-import Puik from '@prestashopcorp/puik'
-import '@prestashopcorp/puik/dist/index.css'
-import App from './App.vue'
-
-const app = createApp(App)
-
-app.use(Puik)
-app.mount('#app')
-```
-
-<p align="right">(<a href="#top">back to top</a>)</p>
-
-#### Dark Mode
-
-We are using the [Tailwind Dark Mode](https://tailwindcss.com/docs/dark-mode).
-
-To use it you just have to write the css dark rules with `dark:<oneRule>` in your css. (I recommand to put your light and dark styles on two lines for greater readability)
-
-##### Exemple
-
-```
-Here we are saying the button background will be blue in light mode and green in dark mode
-
-.my-button {
-  @apply bg-blue;
-  @apply dark:bg-green;
-}
-```
-
-## Using the CSS components
-
-If you don't use VueJS for your application, you can use the CSS only version of our components. It includes all the
-styles used in the VueJs component library.
-
-### Usage
-
-1. Include the CSS in your HTML by using the CDN
-
-```html
-<link
-  rel="stylesheet"
```

Comment:
For the dark mode part i think it's better to not mention it as 99% of the current components doesn't have a dark mode variant
For the CSS components if we aim to replace them by using the web components maybe we should not mention them but if you want to keep the CSS components alongside the web components i can re-add the doc

Reply:
Ok, I agree for the dark mode (currently there's only Button component that uses it). But for the CSS component I think we should keep it. Indeed we'll give the possibility to use Web Component but if someone don't want to use neither Web Component or Vue Component because he uses React for example, he should have the possibility to use CSS Component.

Reply:
As discussed with you @aAmorim27 and @mattgoud : we keep the CSS Component section in the README for now.